Code-the-Dream-School/dd-prac-team1-back
# Feat/meal planner schema pra 153

## One Line Description
allow a user to create, modify, view and delete meal plan

_What this code should do_
Enable users to construct a daily meal by selecting recipes, specifying the day, and indicating the meal slot (breakfast, lunch, or dinner).## Notes

_Any additional info_

## Test Steps

_Instructions on how to test the changes_

/- can be checked either by Postman and Thunder clients in VS. Thunder Client involves sending HTTP requests to the endpoints defined in the mealPlanner_routes and examining the responses to ensure they behave as expected.

Endpoints: 
GET localhost:3000/api/v1/meal-planner - view all meals for a auth user
POST localhost:3000/api/v1/meal-planner - create a meal
PUT localhost:3000/api/v1/meal-planner/:id - modify meal associated with a specific ID.
DELETE localhost:3000/api/v1/meal-planner/:id - delete meal associated with a specific ID.

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed
